dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1685,6 +1772,12 @@ def test_save_and_load_structured(self):
         test_files = ["csv/guns.csv", "csv/iris.csv"]
 
         def _clean_report(report):
+            global_stats = report["global_stats"]
+            if "correlation_matrix" in global_stats and \
+                    report["global_stats"]["correlation_matrix"] is not None:
+                report["global_stats"]["correlation_matrix"] = np.concatenate(
+                    report["global_stats"]["correlation_matrix"].values).tolist()
+
```

Comment:
maybe we should make this a util since we reuse it a lot.

Reply:
you meant the clean_report function? I can do it if we want.

Reply:
indeed

Reply:
brought to utils